project-base/assets/js/frontend/deliveryAddress/deliveryAddress.js

```diff
@@ -0,0 +1,76 @@
+import Register from 'framework/common/utils/register';
+import Ajax from 'framework/common/utils/ajax';
+import Window from '../utils/window';
+import Translator from 'bazinga-translator';
+
+export default class DeliveryAddress {
+    onRemove ($this, deliveryAddress) {
+        const $row = $this.closest('.js-delivery-address-row');
+        const $input = $row.find('.js-delivery-address-input');
+
+        // eslint-disable-next-line no-new
+        new Window({
+            content: Translator.trans('Do you really want to remove this shipping address?'),
+            buttonCancel: true,
+            buttonContinue: true,
+            eventContinue: () => {
+                const deliveryAddressId = $input.val();
+                if (deliveryAddressId > 0) {
+                    const url = $this.data('href') + '/' + deliveryAddressId;
```

Comment:
why dont you just create the right url with the right parameter in the template?

Reply:
fixed